chirp/drivers/ft1d.py

```diff
@@ -2518,37 +2487,37 @@ def apply_WEhemi(cls, setting, obj):
 
     def apply_bt_lat(cls, setting, obj):
         val = setting.value.get_value()
-        val = cls.backtrack_zero_pad(val, 3)
+        val = cls.zero_pad(val, 3)
```

Comment:
Are these related to the stated bug or again just gardening? Seems like `zero_pad()` shouldn't be a method at all, but just a helper function at the top of this file, right? Do any classes actually override this?